Dockerfile

```diff
@@ -140,6 +142,10 @@ RUN wget ${WGET_ARGS} https://github.com/zephyrproject-rtos/sdk-ng/releases/down
 	sh "zephyr-sdk-${ZSDK_ALT_VERSION}-linux-x86_64-setup.run" --quiet -- -d /opt/toolchains/zephyr-sdk-${ZSDK_ALT_VERSION} && \
 	rm "zephyr-sdk-${ZSDK_ALT_VERSION}-linux-x86_64-setup.run"
 
+RUN curl -sSf https://sh.rustup.rs | sh -s -- -y && \
```

Comment:
cant you do that with wget?

Reply:
yes, CI failing is due to source command is not recognized in /bin/sh shell,  I will update them.